source/cost-saving-config.txt

```diff
@@ -6,47 +6,56 @@ Cost-Saving Configurations
 
 .. default-domain:: mongodb
 
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: atlas architecture center
+   :description: Learn about ways optimize your costs while using Atlas.
+
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: onecol
 
-Intro statement
+|service-fullname| offers comprehensive tools and features to
+effectively manage and control an organization's database costs,
+especially as your usage expands. By providing detailed insights and
+user friendly cost optimization features, {+service+} enables you to
+better understand and streamline your spending.  
 
 {+service+} Features and Best Practices for Cost-Saving Configurations
 ----------------------------------------------------------------------
 
-Content here
-
-Examples
---------
-
-The following examples <perform this action> using |service|
-:ref:`tools for automation <arch-center-automation>`.
-
-These examples also apply other recommended configurations, including:
-
-.. tabs::
-
-   .. tab:: Dev and Test Environments
-      :tabid: devtest
-
-      .. include:: /includes/shared-settings-clusters-devtest.rst
-
-   .. tab:: Staging and Prod Environments
-      :tabid: stagingprod
+.. include:: /includes/billing-optimizations.rst
 
-      .. include:: /includes/shared-settings-clusters-stagingprod.rst
+Auto-Scaling {+Clusters+}
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. tabs::
+{+service+} offers {+cluster+} auto-scaling for all tiers except the
+highest tier, specifically under the general and low-CPU classes.
 
-   .. tab:: CLI
-      :tabid: cli
+:ref:`Cluster auto-scaling <cluster-autoscaling>` enables {+clusters+}
+to automatically adjust the tier, storage capacity, or both, in response
+to real-time usage, helping users to maintain optimal performance
+without manual intervention. {+service+} analyzes the CPU utilization
+and memory utilization to determine when and whether to scale the
+{+cluster+} tier up or down.
 
-      Content here
+Users can also specify a range of maximum and minimum {+cluster+} sizes
+that their {+cluster+} can automatically scale to. {+service+} won't
+scale a {+cluster+} if the new tier falls outside a user's specified
+size range or if memory usage would exceed the capacity of the new tier.
+To learn more about {+cluster+} sizes, see
+:ref:`arch-center-cluster-size-guide`.
 
-   .. tab:: Terraform
-      :tabid: Terraform
+Billing Quota Management
```

Comment:
I think this duplicates content in your include from Jeff, so you can remove this section